client/src/main/java/org/spine3/base/StringifierRegistry.java

```diff
@@ -58,17 +59,27 @@ private StringifierRegistry() {
         // Prevent external instantiation of this singleton class.
     }
 
+    @SuppressWarnings("unchecked") // It is OK because the class is checked before the cast.
     static <T> Stringifier<T> getStringifier(Type typeOfT) {
         checkNotNull(typeOfT);
         final Optional<Stringifier<T>> stringifierOptional = getInstance().get(typeOfT);
 
-        if (!stringifierOptional.isPresent()) {
-            final String errMsg =
-                    format("No stringifier registered for the type: %s", typeOfT);
-            throw new MissingStringifierException(errMsg);
+        if (stringifierOptional.isPresent()) {
+            final Stringifier<T> stringifier = stringifierOptional.get();
+            return stringifier;
         }
-        final Stringifier<T> stringifier = stringifierOptional.get();
-        return stringifier;
+
+        if (typeOfT instanceof Class) {
+            final Class<?> aClass = (Class)typeOfT;
```

Comment:
The `isMessage` check should go to some utility class. Please check the existing utility classes in `spine-core`.